src/c-writer.cc

```diff
@@ -1551,7 +1647,11 @@ void CWriter::WriteFuncDeclarations() {
   for (const Func* func : module_->funcs) {
     bool is_import = func_index < module_->num_func_imports;
     if (!is_import) {
-      Write("static ");
+      if (InternalSymbolScope()) {
```

Comment:
I *think* this whole stanza could just be `Write(InternalSymbolScope())` (because `extern` is already the default for a function declaration, so no need for the `if` and explicit `Write("extern ")`).